test/AddCommandTest.py

```diff
@@ -200,7 +200,7 @@ def test_add_dep9(self):
         command = ListCommand.ListCommand(["Bar"], self.todolist, self.out, self.error)
         command.execute()
 
-        self.assertEqual(self.output, "|kbn| {today} Bar p:1 +Project\n|kbn| {today} Bar +Project\n".format(today=self.today))
+        self.assertEqual(self.output, "|kbn| {today} Bar p:1 +Project\n|kbn|   {today} Bar +Project\n".format(today=self.today))
```

Comment:
Most of the tests got fixed in ed2a9dd5e1f4ee2effbff962e342117f81d048d8, but this one was missed at that time. The cause would actaully be an earlier commit, 0877bac73945f765df11ff2715ecbfb99d3732a3.